internal/util/string.go

```diff
@@ -67,3 +67,13 @@ func marksNewWord(i int, input []rune) bool {
 
 	return false
 }
+
+// Deref dereferences string pointer
```

Comment:
The comment is not accurate (any more) since it derefes everything. Also the `string.go` should probably be renamed as well.

Reply:
Thanks, I forgot to adjust the "string" after I expanded it.

Reply:
I couldn't think of a better filename, suggestions are welcome :wink: